spec/models/spree/six_saferpay_payment_spec.rb

```diff
@@ -14,14 +15,29 @@ module Spree
     end
 
     describe "#create_solidus_payment!" do
-      it 'creates a Solidus::Payment with the correct information' do
-        expect(Spree::Payment.count).to eq(0)
-        solidus_payment = payment.create_solidus_payment!
-        expect(Spree::Payment.count).to eq(1)
+      subject(:solidus_payment) { payment.create_solidus_payment! }
+
+      it 'creates a Solidus::Payment' do
+        expect { payment.create_solidus_payment! }.to change { Spree::Payment.count }.from(0).to(1)
+      end
+
+      it 'sets the order' do
```

Comment:
It sets the order of what?

Reply:
changed spec description to be more descriptive